src/ngrest/render/RenderCrud.php

```diff
@@ -48,6 +75,17 @@ public function getView()
         return $this->_view;
     }
     
+    /**
+     * Set crud render view object
+     *
+     * @param \luya\admin\ngrest\render\RenderCrudView|string|array $view Object, classname string or Yii object config array
+     * @since 2.0.0
+     */
+    public function setView($value)
+    {
+        $this->_view = \yii\di\Instance::ensure($value, '\yii\base\View');
```

Comment:
Please add yii\di\Instance to the use statements, we never use "inline" full class names, then everything is perfect on this PR. 👍 thank you very much.